econplayground/templates/assignment/assignment_questions.html

```diff
@@ -145,6 +145,11 @@ <h4>{{question.title}}</h4>
         selectQuestion(e.target, Number(e.target.value));
     });
 
+    $('select.ep-question').select2({
+        class: 'form-select',
```

Comment:
`class` is a reserved keyword in javascript, so using this unquoted may cause issues in certain contexts.  But also, according to select2's docs, this would be specified as `dropdownCssClass` https://select2.org/configuration/options-api  or maybe i'm missing something?